### PR TITLE
[Doc] Add purpose section to README.md and move test section content to sub module.

### PR DESCRIPTION
**Pull request by CaptainVincent**
Should be merged after https://github.com/second-state/rust-ssvm/pull/3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Substrate SSVM Node
 
+The primary objective of this project was to demonstrate how to establish a substrate node that binding SSVM by EVMC.
```
Comment (hydai) [inline README.md]: Our objective should be `to bring EWASM runtime from Ethereum community to Substrate to extend substrate ecosystem`.
You mentioned here is the actual work we've done.

**Review by hydai — CHANGES_REQUESTED** [review]
The objective should be more clear.

File: README.md (hunk 1 above)
Comment (CaptainVincent) [reply README.md]: Replace done.

**Review by hydai — APPROVED** [review]
LGTM

**Pull request merged**